src/Http/Form/SignupForm.php

```diff
@@ -203,7 +203,7 @@ public function validateLastName(): bool
 
     public function validateUrl(): bool
     {
-        if (\preg_match('/^https:\/\/joind\.in\/user\/[a-zA-Z0-9\-_]{1,100}$/', $this->cleanData['url'])
+        if (\preg_match('/^https:\/\/joind\.in\/user\/[a-zA-Z0-9\-_\.]{1,100}$/', $this->cleanData['url'])
```

Comment:
This will also allow users to provide user names which only consist of `.`s, are we ok with that?

Reply:
Well, I just registered at https://joind.in using `.` as a username.

Of course I can sign in, but not view my profile.

I guess we can merge this in, then.